mindmeld/models/helpers.py

```diff
@@ -100,6 +101,50 @@ def create_annotator(config):
         raise KeyError(msg.format(config["annotator_class"]))
 
 
+def create_augmentor(batch_size, config, lang, resource_loader):
+    """Creates an augmentor instance using the provided configuration
+
+    Args:
+        batch_size (int): Batch size for batch processing in augmentation models.
+        config (dict): A model configuration.
+        lang (str): Language for data augmentation.
+        resource_loader (object): Resource Loader object for the application.
+
+    Returns:
+        Augmentor: An Augmentor class
+
+    Raises:
+        ValueError: When model configuration is invalid or required key is missing
+    """
+    if "augmentor_class" not in config:
+        raise KeyError(
+            "Missing required argument in AUGMENTATION_CONFIG: 'augmentor_class'"
+        )
+    try:
+        # Validate configuration input
+        paths = config.get(
+            "paths",
+            [
+                {
+                    "domains": ".*",
+                    "intents": ".*",
+                    "files": ".*",
+                }
+            ],
+        )
+        path_suffix = config.get("path_suffix", "-augment.txt")
+        return AUGMENTATION_MAP[config["augmentor_class"]](
+            batch_size=batch_size,
+            lang=lang,
```

Comment:
Lets standardize to `language`

Reply:
Just made that change everywhere. all 'lang' has been changed to 'language'